packages/block-library/src/heading/edit.native.js

```diff
@@ -38,10 +37,7 @@ const HeadingEdit = ( {
 			identifier="content"
 			tagName={ 'h' + attributes.level }
 			value={ attributes.content }
-			style={ {
-				...style,
-				minHeight: styles[ 'wp-block-heading' ].minHeight,
```

Comment:
I wonder why we had that in the first place. I searched the file history, and unfortunately it comes from [a big merge](https://github.com/WordPress/gutenberg/pull/14620/files). Maybe @Tug knows better if this is safe to change.

Reply:
Yeah it comes from [this PR](https://github.com/WordPress/gutenberg/pull/14404) ([related gb-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/739)).
From what I understand it was used to fix a bug/inconsistency in Aztec related to the placeholder not being displayed when no content was given. I'm not sure if this has been fixed in Aztec, maybe @marecar3 knows? Otherwise @mchowning might have tested it and made sure that was not an issue before removing it?

Reply:
Just double-checked and the placeholders are showing up for headings, paragraphs, etc. for me.